WcaOnRails/app/models/registration.rb

```diff
@@ -25,11 +26,29 @@ class Registration < ActiveRecord::Base
   end
 
   def pending?
-    accepted_at.nil?
+    accepted_at.nil? && deleted_at.nil?
   end
 
   def accepted?
-    !pending?
+    !accepted_at.nil? && deleted_at.nil?
+  end
+
+  def deleted?
+    !deleted_at.nil?
+  end
+
+  def checked_status
+    if accepted?
+      return :a
```

Comment:
(Only if you agree with me and are feeling motivated)

It looks like we have `:a` and `:accepted` symbols scattered throughout this code without any particular system to them. I'd vote for just going with `:accepted`, `:pending`, and `:deleted` everywhere.